homeassistant/components/discovery.py

```diff
@@ -25,6 +25,7 @@
 SERVICE_NETGEAR = 'netgear_router'
 SERVICE_SONOS = 'sonos'
 SERVICE_PLEX = 'plex_mediaserver'
+SERVICE_ONKYO = 'onkyo'
```

Comment:
This should only be added if Netdisco can discover them. I have not seen a PR for that yet. Are you planning this?